.cbmc-batch/source/make_common_data_structures.c

```diff
@@ -61,7 +61,11 @@ void ensure_md_context_has_allocated_members(struct aws_cryptosdk_md_context *ct
     ctx->evp_md_ctx = evp_md_ctx_nondet_alloc();
 }
 
-void ensure_sig_ctx_has_allocated_members(struct aws_cryptosdk_sig_ctx *ctx) {
+struct aws_cryptosdk_sig_ctx *ensure_sig_ctx_has_allocated_members(struct aws_cryptosdk_sig_ctx *ctx) {
```

Comment:
Is it intentional that the implementation of this function accepts an argument, but the declaration (in the header file) and the usage (in the harness) seem to imply that the function is nullary?

Reply:
Nope, this is a mistake. Since this ensure function is actually the same as the one in https://github.com/aws/aws-encryption-sdk-c/pull/606, I've gone ahead and pulled the changes from that PR (also so I don't have to merge later). That fixes this issue!